### PR TITLE
[DI][5.0] Fix suspicious test

### PR DESCRIPTION
**Pull request by ro0NL**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no     
| Deprecations? | no
| Tests pass?   | yes    
| Fixed tickets | #...   
| License       | MIT
| Doc PR        | symfony/symfony-docs#... 

Cleanup of #31944 for 5.0

### CODE REVIEW

File: src/Symfony/Component/DependencyInjection/Tests/Compiler/ValidateEnvPlaceholdersPassTest.php [hunk 1]
```diff
@@ -274,20 +273,6 @@ public function testEnvWithVariableNode(): void
         $this->assertSame($expected, $container->resolveEnvPlaceholders($ext->getConfig()));
     }
 
-    /**
-     * @group legacy
-     */
-    public function testConfigurationWithoutRootNode(): void
```
Comment (ro0NL) [inline src/Symfony/Component/DependencyInjection/Tests/Compiler/ValidateEnvPlaceholdersPassTest.php]: this cannot happen anymore with Config 5.0, in the #31678 the deps are updated .. as such i assumed we dont need this test anymore.

File: src/Symfony/Component/Config/Definition/Exception/TreeWithoutRootNodeException.php [hunk 2]
```diff
@@ -1,19 +0,0 @@
...
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Symfony\Component\Config\Definition\Exception;
-
-/**
- * @author Roland Franssen <franssen.roland@gmail.com>
- */
-class TreeWithoutRootNodeException extends \RuntimeException
```
Comment (ro0NL) [inline src/Symfony/Component/Config/Definition/Exception/TreeWithoutRootNodeException.php]: maybe we forgot to mark this `@internal` in 4.x ... :sweat_smile:

**Review by nicolas-grekas — APPROVED** [review]

File: src/Symfony/Component/Config/Definition/Exception/TreeWithoutRootNodeException.php (hunk 2 above)
Comment (nicolas-grekas) [reply src/Symfony/Component/Config/Definition/Exception/TreeWithoutRootNodeException.php]: let's add it #31944 :)

**Comment by nicolas-grekas** [comment]
Thank you @ro0NL.

**Pull request merged**